docs/breaking-changes.md

```diff
@@ -146,6 +146,15 @@ webContents.setWindowOpenHandler((details) => {
 })
 ```
 
+### Removed: `<webview>` `new-window` event
+
+The `new-window` event of `<webview>` has been removed. There is no direct replacement.
```

Comment:
We should probably give _some_ guidance for how to move forward for apps that depend on this. Mention setWindowOpenHandler and link to docs, but note that there is no automatic forwarding to the embedder?

Reply:
the problem here is that the forwarding cannot be easily replicated on app-level, so mentioning `setWindowOpenHandler` is not that helpful

Reply:
it ought to be possible to figure out which webview an open handler comes from, right...?

Reply:
also the question is whether we need to actually do it in the subsequent `browser-window-created` event instead

Reply:
something like this
main.js
```js
mainWindow.webContents.on('will-attach-webview', (event, webPreferences) => {
  webPreferences.preload = path.join(__dirname, 'preload.js');
});

mainWindow.webContents.on('did-attach-webview', (event, wc) => {
  wc.setWindowOpenHandler(details => {
    wc.send('new-window', details);
    return { action: 'allow' };
  });
});
```

preload.js
```js
const { ipcRenderer } = require('electron');

ipcRenderer.on('new-window', (event, data) => {
  console.log(data);
  ipcRenderer.sendToHost('new-window', data);
});
```

index.js
```js
const webview = document.querySelector('webview')
webview.addEventListener('ipc-message', (event) => {
  if (event.channel === 'new-window') {
    const [details] = event.args;
    console.log(details);
  }
});
```

Reply:
i think you can skip the sendToHost by getting the host of the webview from the main process side, right?

Reply:
But then you would send the IPC normally instead of getting via the webview element and it will have to be passed through the contextBridge

Reply:
@nornagon can we merge this for now and address a better code sample in a follow-up?

Reply:
@miniak no. I'm specifically commenting on the content of this _documentation_ change with a request to improve the documentation. I've given a specific example of code which I think is better.